staging/src/k8s.io/kubectl/pkg/drain/drain.go

```diff
@@ -271,7 +291,9 @@ func (d *Helper) evictPods(pods []corev1.Pod, policyGroupVersion string, getPodF
 				out:                             d.Out,
 			}
 			_, err := waitForDelete(params)
-			if err == nil {
+			if d.DryRunStrategy == cmdutil.DryRunServer {
```

Comment:
is `waitForDelete` going to do anything since we're not actually doing a delete? I suspect we might want to return early on this one, or even reject server dry-run?

Reply:
Ah right. Returned early.